CHANGELOG.rst

```diff
@@ -1,5 +1,51 @@
 .. _changelog:
 
+0.23.0
+------
+
+Renku ``0.23.0`` introduces the Renku Workflow File, a friendlier way to encode and run your data analysis pipelines
+on Renku. This yaml-based file is easy to write and execution is as easy as `renku run workflow.yml`. Add a workflow
```

Comment:
I guess the file is a yaml file and not a yaml-based file; the workflow is yaml-based, the approach is arguably yaml based.

How about:

` Workflows are based on an easy to use yaml file and execution is as easy as...`

Reply:
```suggestion
on Renku. You can write out your workflow in this easy-to-use YAML file, and execute it with `renku run workflow.yml`. Add a workflow
```